### PR TITLE
Feature/df 1879 cleanup generator modul

### PR DESCRIPTION
**Pull request by hamidsoleymani**

### CODE REVIEW

File: csv-generator-app/.gitignore [hunk 1]
```diff
@@ -0,0 +1,32 @@
+HELP.md
```
Comment (birom) [inline csv-generator-app/.gitignore:1]: I think the module do not need an own ".gitignore" config. The files to ignore can be configured in the toplevel ".gitignore" in this project.

File: csv-generator-app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,26 @@
+plugins {
```
Comment (birom) [inline csv-generator-app/build.gradle:1]: In my opinion the plugin section is not needed. The module should be added to the top level ``settigns.gradle file``
eg.:
```
rootProject.name = 'referenceservice'
include 'reference-app'
include 'reference-jobcenter-importer-app'
include 'avam-occupations-importer-app'
include 'csv-generator-app'
```

File: csv-generator-app/gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```
Comment (birom) [inline csv-generator-app/gradle/wrapper/gradle-wrapper.properties:1]: There is no need to use an own gradle wrapper. Please add the module to the ``settings.gradle`` (See my other comment).

File: csv-generator-app/gradlew [hunk 4]
```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env sh
```
Comment (birom) [inline csv-generator-app/gradlew:1]: Please remove this. It is not needed.

File: csv-generator-app/gradlew.bat [hunk 5]
```diff
@@ -0,0 +1,103 @@
+@rem
```
Comment (birom) [inline csv-generator-app/gradlew.bat:1]: Please remove it. It is not needed.

File: csv-generator-app/settings.gradle [hunk 6]
```diff
@@ -0,0 +1 @@
+rootProject.name = 'csv-generator-app'
```
Comment (birom) [inline csv-generator-app/settings.gradle:1]: Please remove it. It is not needed.

File: csv-generator-app/src/main/java/ch/admin/seco/service/CsvGeneratorAppApplication.java [hunk 7]
```diff
@@ -0,0 +1,13 @@
+package ch.admin.seco.service;
```
Comment (birom) [inline csv-generator-app/src/main/java/ch/admin/seco/service/CsvGeneratorAppApplication.java:1]: What does this do? Do we need it?

Comment (hamidsoleymani) [reply csv-generator-app/src/main/java/ch/admin/seco/service/CsvGeneratorAppApplication.java:1]: Do you main the package name ch.admin.seco.service? or just service?

File: csv-generator-app/.gitignore (hunk 1 above)
Comment (hamidsoleymani) [reply csv-generator-app/.gitignore:1]: ok.

File: csv-creator/Readme.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
+### csv-creator
+In package ch.admin.seco.service.generators we have main classes to generate csv file.
+Each class can have Excel files and old csv files as input and an output file.
+Input files are located in src/main/resources/chisco/in
+Output files are located in build/resources/main/chisco/output directory
+Directory occupation-labels contains all actual csv files used in production
+If any new csv files is created in build/resources/main/chisco/output,  it should be copied to
+reference-app/src/main/resources/config/liquibase/changelog/data/occupation-labels
+
```
Comment (fabiangotzen) [inline csv-creator/Readme.md:9]: I would maybe also link/mention the corresponding Jira issues: DF-1620 to give some context.

File: reference-app/build.gradle [hunk 9]
```diff
@@ -76,8 +76,6 @@ dependencies {
     compile "org.mapstruct:mapstruct-jdk8"
     compile "ch.admin.seco.alv:alv-logger-starter"
     compile "ch.admin.seco.alv:alv-jwt-starter"
-    compile group: 'org.apache.poi', name: 'poi', version: '4.1.0'
-    compile group: 'org.apache.poi', name: 'poi-ooxml', version: '4.1.0'
     compile group: 'com.opencsv', name: 'opencsv', version: '4.6'
```
Comment (fabiangotzen) [inline reference-app/build.gradle:79]: Do we still need the opencsv dependency in the reference service?

Comment (hamidsoleymani) [reply reference-app/build.gradle]: no i removed it

**Comment by hamidsoleymani** [comment]
Should be. merged after deployment on monday

**Pull request merged**